### PR TITLE
feat: add support to release helm charts to nexus

### PR DESCRIPTION
**Pull request by rafelbev**

### CODE REVIEW

File: pkg/cmd/helm/release/release_test.go [hunk 1]
```diff
@@ -113,3 +113,50 @@ func TestStepHelmReleaseWithArtifactory(t *testing.T) {
 		t.Logf("ran: %s\n", c.CLI())
 	}
 }
+
+func TestStepHelmReleaseWithNexus(t *testing.T) {
```
Comment (ankitm123) [inline pkg/cmd/helm/release/release_test.go:117]: could we change this to table driven tests? All the 3 test cases are very similar.

**Comment by rafelbev** [comment]
I refactored the common code to make sure we cover all cases the same way

**Comment by msvticket** [comment]
I think you need to merge main into the branch since otherwise the pr jobs stays pending due to this: https://jenkins-x.io/blog/2022/04/22/kubernetes-1.22-tekton/

**Comment by msvticket** [comment]
This PR is apparently abandoned

**Pull request closed without merging**